routes/products.js

```diff
@@ -1,6 +1,6 @@
 const productsRouter = require("express").Router();
 const { getProductById, getAllProducts, createProduct, getProductByName } = require("../db")
-
+//get all products   
 productsRouter.get("/", async (req, res, next) =>{
     try {
         const allProducts = await getAllProducts();
```

Comment:
6: allProducts again to products* ? and notice that product by id is labeled just product.